practice-two/src/components/sections/places.html

```diff
@@ -0,0 +1,116 @@
+<section class="places-section">
+  <div class="container places-container">
+    <h2 class="places-heading">Places</h2>
+
+    <div class="select-wrapper">
+      <select title="location" name="location" class="select-places-locations">
+        <option selected value="san-francisco">San Francisco</option>
+        <option value="south-liliana">South Liliana</option>
+        <option value="blickview">Blickview</option>
+        <option value="new raven">New Raven</option>
+      </select>
+    </div>
+
+    <ul class="list-places-locations">
+      <li class="places-location">
+        <input id="san-francisco" type="radio" name="location" class="location-radio">
+        <label for="san-francisco" class="location">
+          San Francisco
+        </label>
+      </li>
+      <li class="places-location">
+        <input id="south-liliana" type="radio" name="location" class="location-radio">
+        <label for="south-liliana" class="location">
+          South Liliana
+        </label>
+      </li>
+      <li class="places-location">
+        <input id="blickview" type="radio" name="location" class="location-radio">
+        <label for="blickview" class="location">
+          Blickview
+        </label>
+      </li>
+      <li class="places-location">
+        <input id="new-raven" type="radio" name="location" class="location-radio">
+        <label for="new-raven" class="location">
+          New Raven
+        </label>
+      </li>
+    </ul>
+
+    <ul class="places-images">
+      <li>
+        <img
```

Comment:
With image, you should set width and height for parent, then set with: 100%, height: auto for image to avoid break layout when changing the image